group_project_v2/group_project.py

```diff
@@ -594,7 +612,62 @@ def dashboard_detail_view(self, context):
 
         return fragment
 
-    def _build_groups_data(self, workgroups, stage_stats):
+    def __render_user(self, user, stage_stats, filtered_users):
```

Comment:
@jbzdak I don't think double underscore methods are required here; moreover, there aren't any of those in the rest of the codebase (both GPv2 and other edX and related repos, as far as I can tell). So, could you please rename it to have single underscore?